### PR TITLE
[DataFactory] Support map complex values to string in copy activity

### PR DESCRIPTION
**Pull request by shawnxzq**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json [hunk 1]
```diff
@@ -3774,6 +3774,10 @@
           "type": "object",
           "description": "The JSON Path of the Nested Array that is going to do cross-apply. Type: object (or Expression with resultType object)."
         },
+        "mapComplexValuesToString": {
+          "type": "object",
+          "description": "Whether to map complex (arrray and object) values to simple strings in json format. Type: boolean (or Expression with resultType boolean)."
```
Comment (ArcturusZhang) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: A typo here: `arrray`

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json (hunk 1 above)
Comment (ArcturusZhang) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: Why type boolean in description rather just make this field type of boolean? 
Is there any particular reason for `"type": "object"`?

Comment (shawnxzq) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: Fixed, thanks!

Comment (shawnxzq) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: Customer use dynamic value through ADF pipeline parameterization, in this case, the value in the payload is an object which represent the parameterized expression, and the result of the expression has a boolean value.

**Review by pilor — APPROVED** [review]

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**